### PR TITLE
torch implementation for 541 input data

### PR DESCRIPTION
**Pull request by Ryland-Edwards**

### CODE REVIEW

**Comment by Ryland-Edwards** [comment]
wip implementation of a torch model what works with [541,1] data shape, similar to original train.py but does not save a model or make a quantized version yet. Does make a loss figure but not its own it overwrites any file under /Figures/loss_curve.png

looking to add [1081, 1] input shape model, save separate pytorch model, save a quantized model, improve performance as it does run slower than tf version.
